src/promise.rs

```diff
@@ -176,3 +186,33 @@ impl PromiseResolver {
     }
   }
 }
+
+#[derive(Debug, PartialEq)]
+#[repr(C)]
+pub enum PromiseRejectEvent {
+  PromiseRejectWithNoHandler,
+  PromiseHandlerAddedAfterReject,
+  PromiseRejectAfterResolved,
+  PromiseResolveAfterResolved,
+}
+
+#[repr(C)]
+pub struct PromiseRejectMessage(Opaque);
+
+impl PromiseRejectMessage {
+  pub fn get_promise(&self) -> Local<'_, Promise> {
+    unsafe {
+      Local::from_raw(v8__PromiseRejectMessage__GetPromise(self)).unwrap()
+    }
+  }
+
+  pub fn get_event(&self) -> PromiseRejectEvent {
+    unsafe { v8__PromiseRejectMessage__GetEvent(self) }
+  }
+
+  pub fn get_value(&self) -> Local<'_, Value> {
+    unsafe {
+      Local::from_raw(v8__PromiseRejectMessage__GetValue(self)).unwrap()
+    }
+  }
+}
```

Comment:
This one is tricky, it returns `Local`s, but is there's no way to get current isolate/context, whatever to get scope, because there's no such method on `PromiseRejectMessage`.

I took a look at [tests in v8](https://cs.chromium.org/chromium/src/v8/test/cctest/test-api.cc?sq=package:chromium&g=0&l=15570) and they have a helper to obtain current isolate.